src/urllib3/connection.py

```diff
@@ -369,6 +377,58 @@ def request_chunked(
         """
         self.request(method, url, body=body, headers=headers, chunked=True)
 
+    def getresponse(  # type: ignore[override]
+        self,
+        request_url: str,
```

Comment:
Let's make this method only accept keyword arguments (ie `f(*, a, b, c)`) so that position doesn't matter.

Reply:
The `*` is new to me, just googled it, coolio

Reply:
Wrote a [whole article on it](https://sethmlarson.dev/blog/strict-python-function-parameters) if you're interested :P

Reply:
done